gdal/configure.ac

```diff
@@ -2153,6 +2153,65 @@ AC_SUBST(SOSI_ENABLED, $SOSI_ENABLED)
 AC_SUBST(SOSI_LIB, $SOSI_LIB)
 AC_SUBST(SOSI_INC, $SOSI_INC)
 
+dnl ---------------------------------------------------------------------------
+dnl JNI
+dnl ---------------------------------------------------------------------------
+
+AC_ARG_WITH(jni,[  --with-jni[=ARG]        Include JVM JNI support (ARG=Path or no)],,)
```

Comment:
as you mentionned in the checklist, there's already an option to detect the JVM

Reply:
Done.